fv3core/stencils/map_single.py

```diff
@@ -126,6 +126,19 @@ def __call__(
         q4_4: FloatField,
         dp1: FloatField,
     ):
+        """
```

Comment:
Thank you for updating the doc string. Perhaps you can show me where you find this information :)

Reply:
Mostly just from porting the Fortran code. Hopefully the documentation will come out soon and provide a good reference for this